src/openforms/js/components/form/select.js

```diff
@@ -5,14 +5,37 @@ import {localiseSchema} from './i18n';
 
 const Select = Formio.Components.components.select;
 
-const values = [
+const optionsChoices = [
+  {
+    type: 'select',
+    key: 'data.dataSrc',
+    label: 'Data source',
+    description: 'What data to use for the options of this field.',
+    defaultValue: 'manual',
+    data: {
+      values: [
+        {
+          label: 'Manually fill in',
+          value: 'manual',
+        },
+        {
+          label: 'Variable',
+          value: 'variable',
+        },
+      ],
+    },
+    validate: {
+      required: true,
+    },
+  },
   {
     type: 'datagrid',
     input: true,
     label: 'Values',
     key: 'data.values',
     tooltip:
-      'The radio button values that can be picked for this field. Values are text submitted with the form data. Labels are text that appears next to the radio buttons on the form.',
+      'The values that can be picked for this field. Values are text submitted with the form data. ' +
+      'Labels are text that appears next to the radio buttons on the form.',
```

Comment:
I expected this to need a change in some nl.json... but the English string wasn't there. :shrug:

Reply:
This should be added to src/openforms/js/lang/formio/nl.json, but I think that the translations are added at the end of the sprint, that's why I haven't added it :thinking:

Reply:
This wasn't towards this PR, but I expected the string to already exist from previous sprints. Guess not.

Reply:
cool JS-tip: use backticks (`` ` ``) for multiline strings :)